mediacapture-streams/permission-helper.js

```diff
@@ -0,0 +1,10 @@
+async function setPermission(status="granted", scope=["camera", "microphone"]) {
```

Comment:
the "status" argument is never used. Is it a future goal to be able to set setPermission(status=denied, scope=["camera"])? If so, document (or even better, implement).

Reply:
it's used in `GUM-deny.html`